workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/new-regions-interpretations.test.ts.snap

```diff
@@ -7,116 +7,81 @@ Object {
       "AddShape": Object {
         "baseShapeId": "$string",
         "name": "",
-        "shapeId": "shape_6699",
+        "shapeId": "shape_6697",
       },
     },
     Object {
       "AddShape": Object {
         "baseShapeId": "$boolean",
         "name": "",
-        "shapeId": "shape_6700",
+        "shapeId": "shape_6698",
       },
     },
     Object {
       "AddShape": Object {
         "baseShapeId": "$string",
         "name": "",
-        "shapeId": "shape_6701",
+        "shapeId": "shape_6699",
       },
     },
     Object {
       "AddShape": Object {
         "baseShapeId": "$string",
         "name": "",
-        "shapeId": "shape_6702",
+        "shapeId": "shape_6700",
       },
     },
     Object {
       "AddShape": Object {
         "baseShapeId": "$object",
         "name": "",
-        "shapeId": "shape_6708",
-      },
-    },
-    Object {
```

Comment:
@devdoshi here's the shape diff from the snapshot - is this diff related to the diff engine changes that's been going on?

Reply:
yeah I imagine it'll be in flux as we fix some bugs around these areas, we'll be responsible for verifying the final snapshots